.github/workflows/fedora_29.yml

```diff
@@ -85,17 +85,17 @@ jobs:
       build-debian-packages: false
       build-js: false
       build-rpm-packages: true
-      cmake-options: |- 
-            -G "Ninja" \
-            -DENABLE_CPPLINT=OFF \
-            -DENABLE_NCC_STYLE=OFF \
-            -DENABLE_INTEL_NPU=OFF \
-            -DENABLE_TESTS=ON \
-            -DENABLE_STRICT_DEPENDENCIES=OFF \
-            -DENABLE_SYSTEM_OPENCL=ON \
-            -DCPACK_GENERATOR=TGZ \
-            -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
-            -DCMAKE_BUILD_TYPE=Release
+      cmake-generator: 'Ninja'
```

Comment:
What is the benefit to set generator as input argument? We can use CMAKE_GENERATOR if you don't like -G

Reply:
it worked previously, what is the problem with old solution?

Reply:
It worked previously because we were not using `env` which mangles the formatting for the passed strings.